### PR TITLE
added new newsletters

### PR DESCRIPTION
**Pull request by wsbrunson**
Hey, big fan of web dev email newsletters. Added some of my favorite newsletters. Thanks for making this list!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +19,7 @@ A list of amazing Newsletters.
 * [ES6 & ES7 News & Tools](http://esweekly.com/)
 * [Web Operations Weekly](http://webopsweekly.com/)
 * [AngularJS Weekly Newsletter](http://www.ng-newsletter.com/)
+* [ng-newsletter](http://www.ng-newsletter.com)
```
Comment (vredniy) [inline README.md]: I already have this one (one line upper)

File: README.md [hunk 2]
```diff
@@ -21,10 +27,14 @@ A list of amazing Newsletters.
 
 ### CSS
 * [CSS Weekly](http://css-weekly.com/)
+* [CSS-Tricks](https://css-tricks.com) signup is at the bottom of the page.
```
Comment (vredniy) [inline README.md]: Is this newsletter free? I need only free email newsletters

File: README.md [hunk 3]
```diff
@@ -21,10 +27,14 @@ A list of amazing Newsletters.
...
 
 ### Design
 * [Web Design Weekly](https://web-design-weekly.com/)
 * [Product Design Weekly](http://designweekly.atomic.io/). Digital product design news, articles, showcases and resources, delivered straight to your inbox once a week.
+* [Sidebar](http://sidebar.io). Five design-centric posts each day.
+* [Web Desinger News](http://www.webdesignernews.com)
+* [Responsive Design Weekly](http://responsivedesignweekly.com)
```
Comment (vredniy) [inline README.md]: I would like to add it, but the last update was more than a half year ago

**Comment by vredniy** [comment]
@wsbrunson, Thank you for your contributing. I really appreciate it. Can you please fix some links and I'll merge your PR. :+1:

File: README.md (hunk 1 above)
Comment (wsbrunson) [inline README.md]: Ah, yes, didn't notice the links were the same.

File: README.md (hunk 2 above)
Comment (wsbrunson) [inline README.md]: Yes, they have the premium service, but at the bottom of the page is a signup for the free newsletter. I'll try to find a direct link, but so far I haven't seen one.

File: README.md (hunk 3 above)
Comment (wsbrunson) [inline README.md]: Which link specifically? I have received updates from all three in the past week.

**Pull request merged**